pkg/signalmeow/profile.go

```diff
@@ -300,22 +299,6 @@ func decryptString(key *libsignalgo.ProfileKey, encryptedText []byte) (string, e
 	return string(data), err
 }
 
-func encryptString(key libsignalgo.ProfileKey, plaintext string, paddedLength int) ([]byte, error) {
```

Comment:
Probably shouldn't remove this

Reply:
why was it not used?

Reply:
It'll be needed when we want to modify profiles, but we don't do that yet